File: dataset/output/GEMINI_gemini-1.5-pro-001/Task142_GEMINI_gemini-1.5-pro-001.cpp
#include <iostream>

using namespace std;

int gcd(int a, int b) {
  while (b != 0) {
    int temp = b;
    b = a % b;
    a = temp;
  }
  return a;
}

int main() {
  int a, b;
  cout << "Enter first number: ";
  cin >> a;
  cout << "Enter second number: ";
  cin >> b;
  cout << "GCD is: " << gcd(a, b) << endl;
  return 0;
}

File: dataset/output/GEMINI_gemini-1.5-pro-001/task142/Task142_GEMINI_gemini_1_5_pro_001.java
import java.util.Scanner;

public class Task142_GEMINI_gemini_1_5_pro_001 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter first number: ");
        int a = input.nextInt();
        System.out.print("Enter second number: ");
        int b = input.nextInt();
        System.out.println("GCD is: " + gcd(a, b));
    }

    public static int gcd(int a, int b) {
        while (b != 0) {
            int temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }
}